addons/hide-signatures/userscript.js

```diff
@@ -0,0 +1,42 @@
+export default async function ({ addon, global, console, msg }) {
+  // get signatures
+  const posts = document.querySelectorAll(".blockpost");
```

Comment:
WaitForElement

Reply:
I'm not sure if there's consensus on whether waitForElement should be used for forum posts 🤔

Reply:
The script loads and runs when the page is loaded. The posts will already be loaded as they are not added by JS. Addons which touches posts already use `document.querySelector`, not `addon.tab.waitForElement`